chore(curriculum): add two labs to full stack/javascript/loops (#60185)
freeCodeCamp/freeCodeCamp: client/src/pages/learn/full-stack-developer/lab-mutations/index.md

---
title: Introduction to the Mutations Algorithm
block: lab-mutations
superBlock: full-stack-developer
---

## Introduction to the Mutations Algorithm

> In this lab you will practice iterating over two different strings to compare their characters.

Reviewer: Here too:
This lab will focus on iterating over two strings to compare their characters.

> In this lab you will practice iterating over two different strings to compare their characters.

Reviewer: ```suggestion
This lab will focus on iterating over two strings to compare their characters.
```
